astropy/visualization/time.py

```diff
@@ -0,0 +1,218 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+
+import matplotlib.units as units
+from datetime import datetime
+
+from astropy.time import Time
+from astropy import units as u
+from astropy.visualization.wcsaxes.utils import select_step_hour, select_step_scalar
+
+from matplotlib.ticker import MaxNLocator, ScalarFormatter
+
+__all__ = ['time_support']
+
+YMDHMS_FORMATS = ('fits', 'iso', 'isot')
+
+
+def time_support(scale='utc', format='isot', simplify=True):
```

Comment:
Would it be possible to have the default set by the times first used? (say, `scale=None` here)

Reply:
Agree, this will be consistent with ``quantity_support`` - fixed